libs/template/experimental/virtual-scrolling/src/lib/scroll-strategies/autosize-virtual-scroll-strategy.ts

```diff
@@ -303,7 +301,7 @@ export class AutoSizeVirtualScrollStrategy<
     this.viewport = null;
     this.viewRepeater = null;
     this._virtualItems = [];
-    this.resizeObserver?.disconnect();
+    this.resizeObserver?.destroy();
```

Comment:
From what I see the resizeObserver should always be present.

```suggestion
    this.resizeObserver.destroy();
```

Reply:
would be solved by using DI as well